src/particles/beam/BeamParticleContainer.cpp

```diff
@@ -91,6 +91,18 @@ BeamParticleContainer::ReadParameters ()
         "Tilted beams and correlated energy spreads are only implemented for fixed weight beams");
     }
     queryWithParserAlt(pp, "initialize_on_cpu", m_initialize_on_cpu, pp_alt);
+    queryWithParserAlt(pp, "do_spin_tracking", m_do_spin_tracking, pp_alt);
+    if (m_do_spin_tracking) {
+        if (m_injection_type != "from_file") {
+            getWithParserAlt(pp, "initial_spin", m_initial_spin, pp_alt);
+            queryWithParserAlt(pp, "spin_anom", m_spin_anom, pp_alt);
+        }
```

Comment:
```suggestion
        if (m_injection_type != "from_file") {
            getWithParserAlt(pp, "initial_spin", m_initial_spin, pp_alt);
        }
        queryWithParserAlt(pp, "spin_anom", m_spin_anom, pp_alt);
```
The spin anom is used by the beam pusher so it would still need to be read in